pkg/reconciler/ingress/controller.go

```diff
@@ -266,7 +266,7 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 	serviceInformer.Informer().AddEventHandler(controller.HandleAll(
 		controller.EnsureTypeMeta(
 			impl.Tracker.OnChanged,
-			corev1.SchemeGroupVersion.WithKind("Services"),
+			corev1.SchemeGroupVersion.WithKind("Service"),
```

Comment:
Interesting. What about Endpoints and other resources?

Reply:
Endpoints is no problem.